cmd/doctor/check_test.go

```diff
@@ -379,7 +379,7 @@ func TestDoctorCheckProjectTooling(t *testing.T) {
 			mockHookSetup: func(cm *shared.ClientsMock) *shared.ClientFactory {
 				mockDoctorHook := `{"versions": [{"name": "deno", "current": "1.0.0", "message": "Secure runtimes make safer code", "error": {"message": "Something isn't right with this installation"}}]}`
 				mockDoctorScript := hooks.HookScript{Command: "echo checkup"}
-				cm.HookExecutor.On("Execute", hooks.HookExecOpts{Hook: mockDoctorScript}).
+				cm.HookExecutor.On("Execute", mock.Anything, hooks.HookExecOpts{Hook: mockDoctorScript}).
```

Comment:
note: I know a few of these lines are getting longer and could be expanded to multiple lines, but I wanted to keep the changes low since this PR was already getting a little large.